src/views/dashboard/playground/docs/getting-started.md

```diff
@@ -45,74 +61,142 @@ In GreptimeDB, there are three types of columns:
 - `TIME INDEX`: columns of data type TIME
 
 ```sql
-DESC TABLE system_metrics;
+DESC TABLE cpu_metrics;
 ```
 
 ## Add Some Data
 
-Using the `INSERT` statement to easily add data to the table. Below example inserts three rows into the `system_metrics` table.
+Using the `INSERT` statement to easily add data to the table. Below example inserts three rows into the `cpu_metrics` table.
+
+<!-- ts could be timestamp -->
```

Comment:
Maybe changing the `ts` value in the sample data to unix timestamp will be better.